scss/tech.scss

```diff
@@ -0,0 +1,68 @@
+$tech-box-width: 300px;
+$tech-box-height: 450px;
+$tech-box-padding: 15px;
+$tech-box-border-radius: 10px;
+$tech-text-margin: 20px;
+$tech-logo-size: 100px;
+$tech-box-content-displace: 20px;
+
+.tech-box {
+  float:left;
+  height: $tech-box-height + ($tech-box-content-displace * 2);
+  margin-left: 10px;
+  padding: $tech-box-padding;
+  position: relative;
+  text-align: center;
+  width: $tech-box-width + ($tech-box-content-displace * 2);
+
+  &::before {
+    content: '';
+    background-color: $white;
+    border: 1px solid $green;
+    border-radius: $tech-box-border-radius;    
+    height: $tech-box-height;
+    left:0;
+    position: absolute;
+    top:0;
+    width: $tech-box-width;
+  }
+
+  .tech-box-content {
+    align-content: flex-start;
```

Comment:
I think this property is not necessary here. However, did you try setting the `align-items` property?